core/src/workflows/hold_link.rs

```diff
@@ -99,6 +99,12 @@ pub async fn hold_link_workflow(
     // 3. If valid store the entry in the local DHT shard
     await!(add_link(&link_add, &context))?;
     context.log(format!("debug/workflow/hold_link: added! {:?}", link));
+
+    //4. store link_add Entry
+    await!(hold_entry_workflow(&entry_with_header, context.clone()))?;
```

Comment:
Just to clarify does this now mean that the nodes that hold the base also hold the link meta on the base and the link_add entry?

Reply:
Yes.

For the link meta => so that we can get the headers
Link Add Entry => So that we can get the target

Reply:
I think this seems ok. It does mean that LinkAdd entries will be twice as dense in the DHT space when there is sharding so I guess this is a storage vs. retrieval time tradeoff.